### DISCUSSION TITLE
issues with Google maps and autogen grass/vegetation

### DISCUSSION THREAD

**Discussion by umancuso** [derekhe/msfs2020-google-map#49] [General]
Hi, I've tested my hometown airclub (LICP) and a nearby countryside airstrip (LISR), and in both cases google textures are completely fooling MSFS2020 grass&vegetation autogen: the fields around both the airstrips are dull and flat (FSX-like), while with BING textures they are nicely populated with dense grass and vegetation.

Please see the attached pics from LICP (same situation in LISR, I'l post more pics later on):

**BING-LICP:**
![BING-LICP_1](https://user-images.githubusercontent.com/94044442/141101806-a44370dd-c9c7-460f-a4e4-f4bc06ef5d32.jpg)

**GOOGLE-LICP:**
![GOOGLE-LICP_1](https://user-images.githubusercontent.com/94044442/141101978-2ea192d9-4225-4973-b0a5-d7e29eb35ac2.jpg)

**BING-LICP:**
![BING-LICP_2](https://user-images.githubusercontent.com/94044442/141102257-5dd14396-9494-4bf6-84c6-1816b0f02f36.jpg)

**GOOGLE-LICP:**
![GOOGLE-LICP_2](https://user-images.githubusercontent.com/94044442/141102147-f7aeb43e-ef49-437d-95b8-9e6efb0a6c50.jpg)

On the other hand the distant cliffs are true-to-life using Google textures, while default Bing maps have a nasty green hue.
kind regards,
Ugo

**Comment by derekhe** [comment]
Hi, I have a look at the google satellite image and it shows like this: 
https://www.google.com/maps/place/Palermo%E2%80%93Boccadifalco+Airport/@38.1160017,13.3145387,1960m/data=!3m1!1e3!4m5!3m4!1s0x1319ef001006520d:0x468bf7ed6ca67a1a!8m2!3d38.1097894!4d13.3108378

![image](https://user-images.githubusercontent.com/1734077/141119475-a264334b-f5ed-4cfd-a272-41eea9e8fc89.png)

I think it is correct that no grass should be there because it is sand area. I think the google earth image was captured in another season.

And you can see some areas google images are too bright compare to around, these areas has google photogrammetry in google earth

https://earth.google.com/web/@38.11937394,13.30860631,87.35999039a,2783.84458581d,35y,-0.44796475h,65.03470027t,-0r

![image](https://user-images.githubusercontent.com/1734077/141120004-bd5ff274-6df1-41f8-bca4-76febe7817b4.png)

I have already done some tunning to reduce the bright areas using a filter function, but still not so good.
